core/storage/opportunity/gae_models.py

```diff
@@ -52,6 +52,18 @@ def get_deletion_policy():
         """
         return base_models.DELETION_POLICY.KEEP_IF_PUBLIC
 
+    @classmethod
+    def has_reference_to_user_id(cls, unused_user_id):
+        """ExplorationOpportunitySummaryModel doesn't reference any user_id
+        directly.
+        Args:
+            unused_user_id: str. The (unused) ID of the user whose data
+            should be checked.
+        Returns:
```

Comment:
Ditto -- insert single newline above this one.

Reply:
Done.